docs/user.rst

```diff
@@ -25,9 +25,9 @@ If you want, you can organize your templates in one or more folder.
 **How to write the template ?**
 -------------------------------
 
-The `appy framework <http://appyframework.org>`_ is used to interpret the template and render it using the context.
+The `appy framework <https://appyframework.org>`_ is used to interpret the template and render it using the context.
```

Comment:
@Arhell you broke this link... 

Please verify that what you are doing makes sense...

Reply:
Thanks, I'll fix it